app/setup.py

```diff
@@ -160,6 +164,18 @@ def override_url_for():  # pylint: disable=unused-variable
     return application
 
 
+def setup_secure_headers(application):
+    Talisman(
+        application,
+        content_security_policy=CSP_POLICY,
+        content_security_policy_nonce_in=['script-src'],
+        session_cookie_secure=not application.config['EQ_DEV_MODE'],
```

Comment:
We are trying to move away from the `EQ_DEV_MODE` master switch in favour of individual bespoke switches for specific features
So for this we would have something like `EQ_SECURE_SESSION_COOKIE` which would default to True (As we default to production setup) and then override to False in the dev settings

Reply:
Also does this mean we no longer need this? 
https://github.com/ONSdigital/eq-survey-runner/pull/1462/files#diff-03a224d9f3b49bcb5f5cd912c76ca683R321

Reply:
Ah yeah - good call on the specific switch.

It looks like talisman just sets `SESSION_COOKIE_SECURE` internally anyway: https://github.com/GoogleCloudPlatform/flask-talisman/blob/master/flask_talisman/talisman.py#L189

It think it sounds sensible to allow the talisman object to keep control of this so that the secure headers stay handled in a single place. So yeah - let's remove our setting of `SESSION_COOKIE_SECURE`.